radio/src/opentx.cpp

```diff
@@ -1138,8 +1138,6 @@ void checkTHR()
 
   while (1) {
 
-    SIMU_SLEEP(1);
-
     getADC();
```

Comment:
In order to avoid the use of the mutex, we could check if the mixer is running, no?